server/service/src/sync/translations/requisition_line.rs

```diff
@@ -44,6 +44,30 @@ pub struct LegacyRequisitionLineRow {
 
     #[serde(rename = "itemName")]
     pub item_name: String,
+
+    #[serde(rename = "Cust_prev_stock_balance")]
+    pub cust_prev_stock_balance: f64,
```

Comment:
Can you name these with the OMS names and just use serde rename to rename? Thanks